packages/azure-http-specs/specs/service/multi-service/main.tsp

```diff
@@ -0,0 +1,51 @@
+import "@typespec/versioning";
+import "@typespec/http";
+import "@typespec/spector";
+using Versioning;
+using Http;
+using Spector;
+
+namespace Service.MultiService;
+
+@scenarioDoc("""
+  Test that a client can expose operations from multiple services.
+  This scenario demonstrates creating a unified client that provides access to 
+  operations from different services using the @client decorator.
+  
+  ```ts
+  const client = new MultiServiceClient();
+  
+  // Call operation from first service - uses api-version av2
+  client.aTest();
+  
+  // Call operation from second service - uses api-version bv2
+  client.bTest();
+  ```
+  """)
```

Comment:
We probably do not need scenarioDoc here on ServiceA (there is none on ServiceB)?